Portfolio.html

```diff
@@ -4,11 +4,85 @@
     <meta charset="UTF-8">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="Portfolio.css">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Media-Query.css">
```

Comment:
This css file has no content - it is not doing anything - remove?

Reply:
done.